website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -86,53 +86,85 @@ HCP Terraform prioritizes and overwrites conflicting variables according to the
 
 If [prioritized](/terraform/cloud-docs/workspaces/variables#precedence-with-priority-variable-sets), variables in a global variable set have precedence over all other variables with the same key.
 
-### 2. Priority project-scoped variable sets
+### 2. Priority organization-owned variable set scoped to a project
```

Comment:
@rkoron007 I applied your suggestion to these titles, but I'm thinking now it makes more sense to have the scope part come first because the content of the sections focuses more on scope. Maybe something more like `Priority project-scoped variable set owned by an organization`.

Reply:
You make a _really_ good point.  Though, I do kind of worry that without the extra focus on how the ownership affects precedent, the headers become harder to differentiate one from one another.  

For example, the following headers feel easier confuse:
```
2. Priority project-scoped variable set owned by an organization

...and then below...

3. Priority project-scoped variable set owned by a project
```

I always worry people don't read so I try to shift the important details to the left 😂 However, I don't feel super strongly about this, so if you think it'd be better to go by scope go for it!